report/sections/Chapter 1/problem_analysis.tex

```diff
@@ -1,23 +1,30 @@
-The exercises consist of a mix of pair programming and group exercises, which encourages collaboration and sparring.
-In the room during exercise sessions there are teaching assistants and the professor, which will answer the students questions and help if they are stuck on a problem. 
 
-\todo{insert picture of table from PP}
 
-However, with this way of teaching new problems arise.
-Since students use their own local development environments for programming exercises, other students cannot revisit the discussed exercise solutions created by other groups. This makes it difficult to reflect on and learn from other students' solutions.
-Furthermore, it is difficult to verify whether ones own solution is correct and lives up to the requirements of the exercise.
+
+Following the restructuring of the Programming Paradigms course, new problems arose.
+Since students use their own local development environments for programming exercises, other students cannot revisit the discussed exercise solutions created by other groups. 
+This makes it difficult to reflect on and learn from other students' solutions.
+Furthermore, it is difficult to verify whether one's own solution is correct and lives up to the requirements of the exercise.
 In fact, it is not uncommon for people to misunderstand an exercise and therefore implement an incorrect solution.
 
-\subsection{Initial Problem} \label{sec:initial-problem}
-The restructuring of the course has already shown to be an improvement over the former structure, but as mentioned previously it still suffers from a few notable problems related to the execution of the exercise sessions:
+In summary, despite the restructuring of the course being an improvement,
+
+
+% \subsection*{Screen sharing by AAU}
+% Currently AAU uses a room designed by \textit{Center for Digitally Supported Learning} (CDUL) to support interactive learning. 
+% The room is set up with a monitor at each table where a group will sit. The group can then share their screen on the monitor, and the lecturer can then share their own screen or the screens of a group of students to the entire class on the monitors. 
+% Typically the content that is shared with the class are group solutions to exercises, or problems that many had difficulties with.
+% The advantage of this is that many of the common issues are explained to the entire class.
+% The exercises consist of a mix of pair programming and group exercises, which encourages collaboration and sparring.
+% In the room during exercise sessions there are teaching assistants and the professor, which will answer the students questions and help if they are stuck on a problem. 
+
+\todo{insert picture of table from PP}
+
+The restructuring of the course has already shown to be an improvement over the former structure, but as mentioned, it still suffers from a few notable problems related to the execution of the exercise sessions:
```

Comment:
Flow here seems a bit off. We start going into the problem above, and kind of stop. Then we go back into the same thing: describing the issues. I like the list, though. Concise.